### ISSUE TITLE
groupFooter.content.GridView::F_SUM prints "f_sum"

### ISSUE THREAD

**Issue by maxxer** [kartik-v/yii2-grid#1058]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of yii2-grid.
- [x] This is not an usage question. I confirm having gone through and read the [documentation](http://demos.krajee.com/grid) and [demos](http://demos.krajee.com/grid-demo).
- [x] This is not a general programming / coding question. (Those should be directed to the [webtips Q & A forum](http://webtips.krajee.com/questions)).
- [x] I have attempted to find the simplest possible steps to reproduce the issue.
- [ ] I have included a failing test as a pull request (Optional).

## Steps to reproduce the issue

```php
echo GridView::widget([
    'filterModel' => $searchModel,
    'dataProvider' => $dataProvider,
    'columns' => [
        [
            'attribute' => 'company',
            'group' => true,
            'groupFooter' => function ($model, $key, $index, $widget) {
                return [
                    'mergeColumns' => [[0, 1, 2]],
                    'contentOptions' => [
                        0 => ['style' => 'font-weight: bold;'],
                        3 => ['style' => 'text-align: right;'],
                    ],
                    'content' => [
                        0 => $model['company'],
                        3 => GridView::F_SUM,
                    ],
                ];
            },
        ],
        'title',
        [
            'attribute' => 'start_time',
            'format' => 'dateTime',
            'hAlign' => 'right',
        ],
        [
            'attribute' => 'tracking_minutes',
            'format' => 'duration',
            'hAlign' => 'right',
        ],
    ],
]);
```

## Expected behavior and actual behavior

![immagine](https://user-images.githubusercontent.com/240201/225060179-34b1b9dc-5094-43a3-8aa0-99cda47258e0.png)

#### Browsers

- [x] Google Chrome
- [x] Mozilla Firefox
- [ ] Internet Explorer
- [ ] Safari

#### Operating System

- [ ] Windows
- [x] Mac OS X
- [ ] Linux
- [ ] Mobile

#### Libraries

- jQuery version: v3.6.3
- yii2-grid version: c407f8e

## Isolating the problem

- [ ] This bug happens [on the demos page](https://demos.krajee.com/grid-demo)
- [x] The bug happens consistently across all tested browsers
- [ ] This bug happens when using yii2-grid without other plugins.

**Comment by maxxer** [comment]
A friend appointed me the `F_SUM` feature works in javascript, so with plain integer/float values. I tied removing the `format` option from all columns, but still won't sum

**Comment by kartik-v** [comment]
Yes the F_SUM needs a numeric value for the specific Grid DataColumn. You need to ensure that the DataColumn `value` is numeric and you can use the DataColumn `content` property to format the data content for display.

**Comment by zubair1** [comment]
@maxxer you need to add this to your groupFooter return array

```
                                    'contentFormats' => [
                                        3 => ['format' => 'number', 'decimals' => 2],
                                    ],
```
